app/src/main/java/org/coepi/android/cen/CenRepo.kt

```diff
@@ -172,8 +178,8 @@ class CenRepo(private val cenApi: CENApi, private val cenDao: RealmCenDao, priva
     //  Not efficient... It would be best if all observed CENs are loaded into memory
     fun matchCENKey(key : String, maxTimestamp : Int) : List<RealmCen>? {
         // take the last 7 days of timestamps and generate all the possible CENs (e.g. 7 days) TODO: Parallelize this?
-        val minTimestamp = maxTimestamp - 7*24* CENLifetimeInSeconds
-        var possibleCENs = Array<String>(7*24) {i ->
+        val minTimestamp = maxTimestamp - 7*24* 60
+        var possibleCENs = Array<String>(7*24 *(60/CENLifetimeInSeconds)) {i ->
```

Comment:
Can you explain this change?

Reply:
the formula it's wrong, also Sourabh agreeded 
Sourabh Niyogi  1:28 AM
@Bart.Nicolotti on the base64 encode/decode, its should be hex encoding
1:29
On the 7*24*60 you are righ

Reply:
Can you please open a PR with only this change?